locust/argument_parser.py

```diff
@@ -357,7 +357,13 @@ def setup_parser_arguments(parser):
         metavar='LocustClass',
         help="Optionally specify which Locust classes that should be used (available Locust classes can be listed with -l or --list)",
     )
-
+    parser.add_argument(
+        '-B', '--web-auth',
```

Comment:
I don't think we need a short hand `-B` for this option. `--web-auth` should be enough.